package/gluon-core/files/lib/netifd/proto/gluon_wired.sh

```diff
@@ -28,24 +46,27 @@ proto_gluon_wired_setup() {
 
         local meshif="$config"
 
-        local vxlan="$(lua -e 'print(require("gluon.site").mesh.vxlan(true))')"
+        local transitive index vxlan vxpeer6addr
+        json_get_vars transitive index vxlan vxpeer6addr
 
-        local transitive index
-        json_get_vars transitive index
+        # default args
+        [ -z "$vxlan" ] && vxlan=1
```

Comment:
maybe set the old default value using "$(lua -e 'print(require("gluon.site").mesh.vxlan(true))')"? This way it does not break custom interfaces w/o vxlan on sysupgrade.

Reply:
This was never officially supported. Of course, we could do this and (very few) people would benefit. But I think we should not start introducing migration code for unsupported configuration ways.